project-fob/Views/Home/MeetingPageUser.cshtml

```diff
@@ -76,24 +76,35 @@
                 button.draw(dataButton, optionsButton);
                 google.visualization.events.addListener(button, 'select', clickHandler);
             }, 50);
-            
+
         }
 
         if (!isItDrawing) {
             runUpdate();
             isItDrawing = true;
         }
+
+
         var errorCounter = 0;
         function runUpdate() {
-            setTimeout(runUpdate, 10000); //Run update every 10 second
+            setTimeout(runUpdate, 5000); //Run update every 5 second
             $.ajax({
                 type: "POST",
                 url: '@Url.Action("ImStillHere", "Attendee")',
+                data: { meetingString: "@ViewBag.meetingId"},
                 success: function (result) {
-                    var tmp = result + "";
-                    if (tmp.length != 0) {
-                        alert(result);
+                    var returnedvalue = parseInt(result);
+                    if (topicNumber != returnedvalue) {
+                        if (returnedvalue != -1) {
+                            alert("Topic changed to #" + returnedvalue);
```

Comment:
What's the topic change alert for?

Reply:
I put it there for testing purpose in case I want to give some feedback to the user in case the meeting topic changed... But will get rid of the "alert" as it's a bit annoying in a real world to click ok every single time.